### DISCUSSION TITLE
Update Error  No base path (filesdir)

### DISCUSSION THREAD

**Discussion by Taste-IT** [tine20/tine20#7294] [General]
Hello,
i want to upgrade to latest version from version before. While upgrading from CLI i got
Tinebase_Exception_Backend: No base path (filesdir) configured or path not writeable in /web/Tinebase/FileSystem.php:135

My configuration looks good, permissions seems to be fine. Any idea?
thx

**Comment by pschuele** [comment]
hi,

- do you have a full stacktrace of the problem?
- do you have a (writable) "filesdir" configured in you config.inc.php?

Philipp
